Resources/Storage/EchoStorage.py

```diff
@@ -0,0 +1,118 @@
+""" :mod: EchoStorage
+    =================
+
+    .. module: python
+    :synopsis: Echo module based on the GFAL2_StorageBase class.
+"""
+
+
+import os
+
+# from DIRAC
+from DIRAC.Resources.Storage.GFAL2_StorageBase import GFAL2_StorageBase
+from DIRAC import gLogger, S_ERROR, S_OK
+
+
+class EchoStorage(GFAL2_StorageBase):
+  """ .. class:: EchoStorage
+
+    Interface to the Echo storage.
+
+    This plugin will work with both gsiftp and root protocol.
+    According to the RAL admins, Echo best use is done:
+      * using gsiftp for all WAN transfers, and LAN write.
```

Comment:
Another sphinx syntax thing:
The bullet list should be separated by an empty line, but indented to the same level as the preceding paragraph.